forte_wrapper/__init__.py

```diff
@@ -0,0 +1,9 @@
+from forte_wrapper.allennlp import *
```

Comment:
I am actually thinking to leave out this file. This will require the dependencies of all these sub-tools to be met. In forte-wrapper design, we consider users to use one tool at a time, or one tool in one environment. It is very hard to use all the tools at the same time.